packages/chrome/src/elements/nav/NavItem.tsx

```diff
@@ -24,7 +24,7 @@ interface INavItemProps extends HTMLAttributes<any> {
 }
 
 /**
- * Accepts all `<button>` attributes and events
+ * @extends ButtonHTMLAttributes<HTMLButtonElement>
```

Comment:
This should be `HTMLAttributes<HTMLDivElement>` and the `extends` above should be improved by replacing `<any>` with `<HTMLDivElement>`.